CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# How to Contribute
+
+This document outlines some of the conventions on development workflow.
+
+## Contribution Flow
+
+This is a rough outline of what a contributor's workflow looks like:
+
+- Create a topic branch from where you want to base your work (usually master).
+- Make commits of logical units.
+- Make sure your commit messages are in the proper format (see below).
+- Push your changes to a topic branch in your fork of the repository.
+- Make sure the tests pass, and add any new tests as appropriate.
+- Submit a pull request to the original repository.
```

Comment:
Do we really need this part? This is more related to how to work with GH forks rather than DTK related.